http-clients/netty-nio-client/src/main/java/software/amazon/awssdk/http/nio/netty/internal/SdkSocketOptions.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2010-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.http.nio.netty.internal;
+
+import io.netty.channel.ChannelOption;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+import software.amazon.awssdk.annotations.SdkInternalApi;
+
+@SdkInternalApi
+public class SdkSocketOptions {
```

Comment:
Can this be package private?

Reply:
Sorry for the confusion, since it's in the .internal package, it actually should be `@SdkInternalApi`. I meant removing the `public` qualifier but we can' do that :)